src/BridgedRoom.ts

```diff
@@ -379,9 +379,10 @@ export class BridgedRoom {
         };
 
         const reply = await this.findParentReply(message);
+        let parentStoredEvent;
```

Comment:
I figured out that this has the type `EventEntry` but I don't know how to reference that type (linter was saying it was undefined)

Reply:
```suggestion
        let parentStoredEvent: EventEntry;
```

also import `import { EventEntry } from "./datastore/Models";`

Reply:
Having done this my linter tells me, `Type EventEntry | null is not assignable to type EventEntry` on the line below. What do I do to allow it to do both?